cli/connect/console.go

```diff
@@ -276,7 +277,27 @@ func getExecutor(console *Console, connectCtx ConnectCtx) (func(string), error)
 		console.livePrefixEnabled = false
 	}
 
-	return executor, nil
+	signalable_executor := func(in string) {
+		// Signal handler.
+		handleSignals := func(console *Console, stop chan struct{}) {
+			sigCh := make(chan os.Signal, 1)
```

Comment:
```suggestion
			sig := make(chan os.Signal, 1)
```
I think we could rename this.

Reply:
Done.